package.json

```diff
@@ -1,87 +1,94 @@
-    "branches": 100
-  },
-  "prettier": {
-    "tabWidth": 2,
-    "useTabs": false,
-    "singleQuote": true,
-    "bracketSpacing": false,
-    "semi": false,
-    "trailingComma": "none"
-  },
-  "xo": {
-    "prettier": true,
-    "esnext": false,
-    "rules": {
-      "eqeqeq": [
-        "error",
-        "always",
-        {
-          "null": "ignore"
+    "name": "unist-util-remove",
+    "version": "2.0.1",
+    "description": "unist utility to remove nodes from a tree",
+    "license": "MIT",
+    "keywords": [
+        "unist",
+        "unist-util",
+        "util",
+        "utility",
+        "ast",
+        "tree",
+        "node",
+        "cascade",
+        "delete",
+        "mutable",
+        "remove",
+        "squeeze",
+        "strip"
+    ],
+    "repository": "syntax-tree/unist-util-remove",
+    "bugs": "https://github.com/syntax-tree/unist-util-remove/issues",
+    "funding": {
+        "type": "opencollective",
+        "url": "https://opencollective.com/unified"
+    },
+    "author": "Eugene Sharygin <eush77@gmail.com>",
+    "contributors": [
+        "Eugene Sharygin <eush77@gmail.com>",
+        "Titus Wormer <tituswormer@gmail.com> (https://wooorm.com)"
+    ],
+    "files": [
+        "index.js",
+        "types/index.d.ts"
+    ],
+    "types": "types/index.d.ts",
+    "ignores": [
+        "types"
+    ],
+    "dependencies": {
+        "unist-util-is": "^4.0.0"
+    },
+    "devDependencies": {
+        "dtslint": "^4.0.0",
+        "nyc": "^15.0.0",
+        "prettier": "^2.0.0",
+        "remark-cli": "^9.0.0",
+        "remark-preset-wooorm": "^8.0.0",
+        "tape": "^5.0.0",
+        "unist-builder": "^2.0.0",
+        "xo": "^0.35.0"
+    },
+    "scripts": {
+        "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
+        "test-api": "node test",
+        "test-coverage": "nyc --reporter lcov tape test.js",
+        "test-types": "dtslint types",
+        "test": "npm run format && npm run test-coverage && npm run test-types"
+    },
+    "nyc": {
+        "check-coverage": true,
+        "lines": 100,
+        "functions": 100,
+        "branches": 100
+    },
+    "prettier": {
+        "tabWidth": 2,
+        "useTabs": false,
+        "singleQuote": true,
+        "bracketSpacing": false,
+        "semi": false,
+        "trailingComma": "none"
+    },
+    "xo": {
+        "prettier": true,
+        "esnext": false,
+        "rules": {
+            "eqeqeq": [
+                "error",
+                "always",
+                {
+                    "null": "ignore"
+                }
+            ],
+            "no-eq-null": "off",
+            "unicorn/explicit-length-check": "off"
         }
-      ],
-      "no-eq-null": "off",
-      "unicorn/explicit-length-check": "off"
+    },
+    "remarkConfig": {
+        "plugins": [
+            "preset-wooorm"
+        ]
     }
-  },
-  "remarkConfig": {
-    "plugins": [
-      "preset-wooorm"
-    ]
-  }
-}
+}
```

Comment:
indent and missing newline at end of file seems unexpected.
it doesn't seem to match with prettier?

Reply:
Indeed, .prettierignore is ignoring *.json
I guess it takes my default then (which is 4 spaces)

Reply:
That’s weird! Feel free to remove `*.json` from `.prettierignore` if that fixes things for you

Reply:
> I guess it takes my default then (which is 4 spaces)

We also have an `.editorconfig`, preferring 2 spaces. Most text editors can take those into account. So you might be able to configure your editor to solve it too

Reply:
1. Installed editorconfig globally `npm install -g editorconfig`
2. Added the editorconfig extension to Visual Studio Code
3. Disable extension JS-CSS-HTML Formatter (which was overriding formatting on save)
Works much better now.